source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+|    6 | Walltime requested (in ``HH:MM:SS`` format). See the table below for |
+|      | other formats.                                                       |
++------+----------------------------------------------------------------------+
+|    7 | Partition (queue) to use                                             |
++------+----------------------------------------------------------------------+
+|    8 | Number of compute nodes requested                                    |
++------+----------------------------------------------------------------------+
+|    9 | Blank line                                                           |
++------+----------------------------------------------------------------------+
+|   10 | Change into the run directory                                        |
++------+----------------------------------------------------------------------+
+|   11 | Copy the input file into place                                       |
++------+----------------------------------------------------------------------+
+|   12 | Run the job ( add layout details )                                   |
++------+----------------------------------------------------------------------+
+|   13 | Copy the output file to an appropriate location.                     |
++------+----------------------------------------------------------------------+
+
+.. _frontier-interactive:
+
+Interactive Jobs
+----------------
+
+Most users will find batch jobs an easy way to use the system, as they allow
+you to "hand off" a job to the scheduler, allowing them to focus on other tasks
+while their job waits in the queue and eventually runs. Occasionally, it is
+necessary to run interactively, especially when developing, testing, modifying
+or debugging a code.
+
+Since all compute resources are managed and scheduled by Slurm, it is not
+possible to simply log into the system and immediately begin running parallel
+codes interactively. Rather, you must request the appropriate resources from
+Slurm and, if necessary, wait for them to become available. This is done
+through an "interactive batch" job. Interactive batch jobs are submitted with
+the :command:`salloc` command. Resources are requested via the same options
+that are passed via ``#SBATCH`` in a regular batch script (but without the
+``#SBATCH`` prefix). For example, to request an interactive batch job with the
+same resources that the batch script above requests, you would use ``salloc -A
+ABC123 -J RunSim123 -t 1:00:00 -p batch -N 1024``. Note there is no option for
+an output file...you are running interactively, so standard output and standard
+error will be displayed to the terminal.
 
-Another feature of LMOD is swapping or unloading an upstream
-dependency. In these cases, any downstream module will still be loaded
-but inactivated.
+.. warning::
 
-.. code-block:: shell
+    Indicating your shell in your :command:`salloc` command is NOT recommended
```

Comment:
Do Not indicate your shell in your salloc command 
(for example, salloc ... /bin/bash). This would cause your compute job to start on a login node by default, rather than automatically moving you to a compute node.

Reply:
Fixed in a new push.